lib/PuppeteerSharp.Tests/Puppeteer/ExecutablePathTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.IO;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Puppeteer
+{
+    public class ExecutablePathTests
```

Comment:
You need to make this class inherit from `PuppeteerBaseTest` and add the attribute `[Collection("PuppeteerLoaderFixture collection")]`

The problem we have here is that, unluckily, this is the first test running, so we need to download Chromium first, and `PuppeteerBaseTest` does that job for us.

Reply:
@Meir017 do you think you will be able to implement this by tomorrow? 
If not I can merge it and implement it in another PR.